manifest.yaml

```diff
@@ -0,0 +1,20 @@
+---
+applications:
+- name: keycloak
+  routes:
+    - route: identity-idva-keycloak-((ENVIRONMENT_NAME)).apps.internal
+  instances: 1
+  health-check-type: process
+  memory: 1.2G
+  disk_quota: 2G
```

Comment:
These are very specific, and there's no README to explain why the values are what they are. Why are we deploying 1 instead of 2 for HA? Why 1.2GB memory? Does Keycloak really use that much?

Reply:
HA requires additional configuration. It does require about 1G